src/common/utils/auth/authOrRedirect.ts

```diff
@@ -6,11 +8,10 @@ interface Params {
 }
 
 /**
- * Should rename this to "handleProtectedRoute" or something similar
  * If user is not logged in, redirect to the login page
  * If user is logged in, but email is not verified, redirect to the verify email page
  */
-export async function getUserOrServerRedirect(params?: Params) {
+export async function authOrRedirect(params?: Params) {
```

Comment:
Nice. Do we want to still return the user from here? Maybe we should make those single responsibility utils.

Reply:
yeah i agree, no need to return user